docs/changehistory/2.6.0.md

```diff
@@ -19,7 +19,7 @@ The ESLint configuration was moved from `@bentley/build-tools` to a new ESLint p
   "lint": "eslint ./src/**/*.{ts,tsx} 1>&2",
 },
 "devDependencies": {
-  "eslint": "^6.8.0",
+  "eslint": "^7.11.0",

```

Comment:
Can you revert this change?  2.6.0 won't support 7.x :)

Reply:
Great catch! Bad find all and replace...
I'll come back and resolve these when our official 4.0 ver is out